km_api/know_me/urls.py

```diff
@@ -10,6 +10,8 @@
     url(r'^emergency-items/(?P<pk>[0-9]+)/$', views.EmergencyItemDetailView.as_view(), name='emergency-item-detail'),   # noqa
     url(r'^media-resources/(?P<pk>[0-9]+)/$', views.MediaResourceDetailView.as_view(), name='media-resource-detail'),   # noqa
     url(r'^items/(?P<pk>[0-9]+)/$', views.ProfileItemDetailView.as_view(), name='profile-item-detail'),                 # noqa
+    url(r'^items/(?P<pk>[0-9]+)/list-entries/$', views.ListEntryListView.as_view(), name='list-entry-list'),            # noqa
```

Comment:
If this `pk` is the key for a profile item, then the filter backend should be filtering for profile items, not list content.